java/src/main/java/ai/rapids/cudf/TableWithMeta.java

```diff
@@ -0,0 +1,2 @@
+package ai.rapids.cudf;public class TableWithMeta {
```

Comment:
```suggestion
package ai.rapids.cudf;

public class TableWithMeta {
```

Reply:
Wait, I don't see this class is used anywhere?

Reply:
oops

Reply:
Good catch!